verification/src/http_server/handlers/verification/solidity/types.rs

```diff
@@ -64,12 +64,15 @@ impl TryFrom<MultiPartFiles> for CompilerInput {
 
 #[derive(Clone, Debug, Deserialize)]
 pub struct StandardJson {
```

Comment:
do we really need this struct? can't we just make 
```json
"standard_json": "{json}"
```
and not 
```json
"standard_json": {
 "input": "{json}"
},
```
?
Is this future proofing?

Reply:
we need StandardJson because `T` in `VerificationRequest<T>` should be struct with fields